scripts/pylint_extensions.py

```diff
@@ -1384,32 +1384,32 @@ def process_module(self, node):
                 is_docstring = False
 
 
-class SpaceBelowFileOverviewChecker(checkers.BaseChecker):
-    """Checks if there is an empty line between file overview and imports.
-       Note: The check assumes that all files have a file overview, this
-       assumption is justified because Pylint has an inbuilt check for missing
-       file overview.
+class BlankLineBelowFileOverviewChecker(checkers.BaseChecker):
+    """Checks if there is a single empty line below the fileoverview docstring.
+    Note: The check assumes that all files have a file overview. This
+    assumption is justified because Pylint has an inbuilt check for missing
+    file overview.( Name of this check is missing-docstring.)
```

Comment:
There should be a space before '(' and no space after that character. Also, "Name" should be "The name".

You could alternatively just say: "... because Pylint has an inbuilt check (missing-docstring) for missing file overviews." and drop the last sentence.

Reply:
done